tests/test_bcrypt.py

```diff
@@ -494,3 +498,41 @@ def test_2a_wraparound_bug():
         )
         == b"$2a$04$R1lJ2gkNaoPGdafE.H.16.1MKHPvmKwryeulRe225LKProWYwt9Oi"
     )
+
+
+@pytest.mark.thread_unsafe()
```

Comment:
Why is this thread unsafe?

Reply:
Probably better to use `pytest.mark.parallel_threads(1)` instead of `thread_unsafe()`. They're equivalent but the former communicates better what I want. Since the test is already spawning a bunch of threads and is slow it seemed incorrect to me to also run it under `pytest-run-parallel`.